examples/CMS Woodlawn Campus.xml

```diff
@@ -286,7 +286,7 @@
             </ScenarioType>
             <LinkedPremises>
               <Building>
-                <LinkedFacilityID IDref="Facility22"/>
+                <LinkedBuildingID IDref="Facility22"/>
```

Comment:
Didn't want to change the id of the building in this example?

Reply:
thanks for catching this. Just updated it.